cg/apps/sequencing_metrics_parser/__init__.py

```diff
@@ -0,0 +1,18 @@
+from .parsers.bcl2fastq import (
```

Comment:
I usually add the complete path for clarity, so `cg.apps.sequencing_metrics`...

Reply:
I think I prefer using . (this directory). Lets decide on a standard, it seems like it is done differently in our repos

Reply:
I prefer absolute imports because:
"they are quite clear and straightforward. It is easy to tell exactly where the imported resource is, just by looking at the statement. Additionally, absolute imports remain valid even if the current location of the import statement changes. "
And it also explicitly recommended in PEP8

Reply:
We should create a document where we record these coding standards